osxkeychain.go

```diff
@@ -25,12 +25,13 @@ import (
 // All string fields must have size that fits in 32 bits. All string
 // fields except for Password must be encoded in UTF-8.
 type GenericPasswordAttributes struct {
-	ServiceName string
-	AccountName string
-	Password    string
+	ServiceName         string
+	AccountName         string
+	Password            []byte
+	TrustedApplications []string
```

Comment:
Also add a comment saying that TrustedApplications refers to other programs, and that the current one is always trusted